bower.json

```
@@ -6,7 +6,7 @@
   "issues": "https://github.com/adaptlearning/adapt_framework/issues/new?title=contrib-bookmarking%3A%20please%20enter%20a%20brief%20summary%20of%20the%20issue%20here&body=please%20provide%20a%20full%20description%20of%20the%20problem,%20including%20steps%20on%20how%20to%20replicate,%20what%20browser(s)/device(s)%20the%20problem%20occurs%20on%20and,%20where%20helpful,%20screenshots.",
   "displayName" : "Bookmarking",
   "extension" : "bookmarking",
-  "description": "A bookmarking extension",
+  "description": "Keeps record of the learner's current location within the course so that when they quit and relaunch they may be given the option to be returned to that same location",
```

Apparently the limit for the description is 140 chars; this is 166. Could you change it to...
```suggestion
  "description": "An extension that tracks the learner's location in the course so that it can return them to that location in a subsequent session",
```